OpenRA.Mods.Common/Traits/EngineerRepair.cs

```diff
@@ -18,8 +19,11 @@
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Can instantly repair other actors, but gets consumed afterwards.")]
-	class EngineerRepairInfo : ITraitInfo
+	class EngineerRepairInfo : ConditionalTraitInfo
 	{
+		[Desc("Uses the \"EngineerRepairable\" trait to determine whether or not we should clone a produced unit.")]
```

Comment:
...clone a produced unit?

Reply:
Copy pasted from Cloneable, missed that part.